### ISSUE TITLE
能否增加 撤回消息

### ISSUE THREAD

**Issue by vD1S** [anhkgg/SuperWeChatPC#12]
这个功能很实用 ☺

**Comment by vD1S** [comment]
使用SDK发送消息后 返回msgid ，然后可以按msgid去撤回对于信息。

**Comment by buxiu** [comment]
大佬能不能在拦截撤回消息成功之后，加一个提示说明这条是被撤回的消息，不然在实际使用过程中，就不知道这条消息是不是被撤回过的

**Comment by cuishuoguo** [comment]
是的 大佬能否增加这个提示呢 xxx 撤回了某条消息 卡片形式展示   。为啥会有这种想法呢  因为之前防撤回 别人撤回了 还是敏感话题 ，但是我不知道那条撤回了 继续说那个话题，结果就出很大的麻烦了……感激不尽，希望大佬可以考虑下呢

**Comment by anhkgg** [comment]
@buxiu @cuishuoguo 好的

**Comment by cuishuoguo** [comment]
> @buxiu @cuishuoguo 好的

哈哈哈，谢谢大佬，大佬威武。可以考虑设计一个卡片形式，最上方提示是拦截到的撤回消息 发送人是谁 撤回的消息是啥 。大佬威武 大佬辛苦啦，祝大佬发财 嘿嘿

**Comment by anhkgg** [comment]
![revokemsg](https://user-images.githubusercontent.com/9443285/54004774-cb657c80-4191-11e9-9a28-d19076a314b0.png)

**Comment by buxiu** [comment]
@anhkgg 感谢大佬

**Comment by cuishuoguo** [comment]
感谢大佬

**Comment by cuishuoguo** [comment]
大佬啥时候加入新的版本下载里面去呀，我来更换版本

**Comment by buxiu** [comment]
@anhkgg 大佬，图片和表情撤回后显示不出来，只有文字的内容才能显示出来，如果大佬有空的话，希望可以再优化一下，谢谢

**Comment by cuishuoguo** [comment]
> @anhkgg 大佬，图片和表情撤回后显示不出来，只有文字的内容才能显示出来，如果大佬有空的话，希望可以再优化一下，谢谢

新版本带撤回显示的你在哪里下载的？我咋没看到啊。烦请大哥说下

**Comment by buxiu** [comment]
@cuishuoguo 2小时前更新了，你再下载看看

**Comment by anhkgg** [comment]
![2](https://user-images.githubusercontent.com/9443285/54018234-177ce500-41c3-11e9-9185-cb4892cd1bcb.png)
图片好像不行，表情可以，我后面在看看

**Comment by buxiu** [comment]
@anhkgg 
好像微信自带表情才行，用户自定义的表情不可以
